sdk/nodejs/dagger/api/api.ts

```diff
@@ -0,0 +1,309 @@
+import { Engine, GraphQLClient, gql } from "../../dist/index.js";
```

Comment:
what is the difference between `api/client.ts` and `dagger/api/api.ts`?

Reply:
It some remnants from the previous version of the API that didn't get deleted, it seems. Will clean it up.

Reply:
We're gonna clean them up after the release.